parquet-tools/src/main/java/parquet/tools/command/CatCommand.java

```diff
@@ -52,7 +69,11 @@ public void execute(CommandLine options) throws Exception {
       PrintWriter writer = new PrintWriter(Main.out, true);
       reader = new ParquetReader<SimpleRecord>(new Path(input), new SimpleReadSupport());
       for (SimpleRecord value = reader.read(); value != null; value = reader.read()) {
-        value.prettyPrint(writer);
+        if (options.hasOption('j')) {
+          value.prettyPrintJson(writer);
```

Comment:
Sure I'll go for the first approach and factor this out to a util class then?
Don't have a lot of time this week so it might take a bit longer.